recipe/meta.yaml

```diff
@@ -6,21 +6,22 @@ package:
     version: {{ version }}
 
 source:
-    url: https://github.com/UV-CDAT/cdutil/archive/v{{ version }}.tar.gz
+    url: https://github.com/CDAT/cdutil/archive/v{{ version }}.tar.gz
     sha256: 2430f02d4e840d16e49af7ab6053f85bf4c55c82ea5fa36790948083be2f2465
 
 build:
-  number: 1
+  number: 2
   skip: True  # [win]
-  script: python setup.py install
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
 
 requirements:
-  build:
-    - python
+  host:
+    - python {{python}}
     - setuptools
     - cdat_info
```

Comment:
We could probably get rid of `setuptools` and `cdat_info` as requirements, but I haven't tried that myself.

Reply:
We need setuptools and cdat_info for `cdutil`
https://github.com/CDAT/cdutil/blob/master/setup.py
https://github.com/CDAT/cdutil/blob/master/cdutil/__init__.py